### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by phgross**
- Move translation of strings profile XMLs (actions.xml, ftis etc.) which has been moved to the opengever.core-profile recently, to a separate domain `opengever.core`
- Update all translations files
- Improve /fix some translations strings according to our translators feedback.

### CODE REVIEW

File: opengever/core/locales/de/LC_MESSAGES/opengever.core.po [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+msgid "Create Disposition"
+msgstr "Aussonderungsangebot erstellen"
+
+#: ./opengever/core/profiles/default/actions.xml
+msgid "Create Task"
+msgstr "Aufgabe erstellen"
+
+#: ./opengever/core/profiles/default/actions.xml
+msgid "Delete"
+msgstr "Supprimer"
```
Comment (deiferni) [inline opengever/core/locales/de/LC_MESSAGES/opengever.core.po]: this is french 🤔

File: opengever/core/locales/de/LC_MESSAGES/opengever.core.po [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+msgid "Submit additional documents"
+msgstr "Zusätzliche Anhänge einreichen"
+
+#: ./opengever/core/profiles/default/types/opengever.meeting.submittedproposal.xml
+msgid "Submitted Proposal"
+msgstr "Eingereichten Antrag"
+
+#: ./opengever/core/profiles/default/types/opengever.task.task.xml
+msgid "Task"
+msgstr "Tâche"
```
Comment (deiferni) [inline opengever/core/locales/de/LC_MESSAGES/opengever.core.po]: this is also french.

File: opengever/core/locales/de/LC_MESSAGES/opengever.core.po [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+msgid "untrashed"
+msgstr "Reaktivieren"
+
+#: ./opengever/core/profiles/default/actions.xml
+msgid "with comment"
+msgstr "Mit Kommentar"
+
+#: ./opengever/core/profiles/default/actions.xml
+msgid "without comment"
+msgstr "Ohne Kommentar"
```
Comment (deiferni) [inline opengever/core/locales/de/LC_MESSAGES/opengever.core.po]: shouldn't this string be deleted from another file?

**Comment by phgross** [comment]
Some of the message strings are still used in other packages, therefore they're not removed in other files. 

Because the bin/i18n-build checks only for translations inside the subpackage, I would let the duplicated for now.

**Comment by phgross** [comment]
@deiferni updated ...

**Pull request merged**